arrocke/gloss-translation
# feat: footnotes

## What has changed
- Added footnotes section to Notes tab
- Created `Footnote` schema
- Rename paths: 
  - `/languages/{code}/verses/{verseId}/notes` --> `/languages/{code}/verses/{verseId}/translator-notes`
  - `/languages/{code}/words/{wordId}/note` --> `/languages/{code}/words/{wordId}/note`
- Created paths:
  - GET: `/languages/{code}/verses/{verseId}/footnotes` 
  - PATCH: `/languages/{code}/words/{wordId}/footnote`

## Connected Issues
closes #311


## QA Steps


1. Login as Admin.
1. Open the sidebar for a word and go to the Notes tab.
2. Confirm that there is a "Footnotes" section in the panel with rich text editor.
3. Edit the footnotes.
4. Click outside the editor. Confirm that the footnote's content was saved and the timestamp was updated.
5. Switch to a different word and switch back to the original word. Confirm that the previous changes persisted.
6. Reload the page. Again, confirm that the previous changes persisted.

## Post-Deployment
